pkg/tfbridge/provider_test.go

```diff
@@ -1142,9 +1173,9 @@ func TestCheckConfig(t *testing.T) {
 		require.NoError(t, err)
 		require.NoError(t, err)
 		require.Equal(t, 1, len(resp.Failures))
-		require.Equal(t, "could not validate provider configuration: "+
-			"Invalid or unknown key. Check `pulumi config get testprovider:cofnigValue`. "+
-			"Did you mean `testprovider:configValue`?",
+		require.Equal(t, "`testprovider:cofnigValue` is not a valid configuration key for the testprovider provider. "+
+			"Did you mean `testprovider:configValue`? "+
+			"If the key is not intended for the provider, please choose a different namespace from `testprovider:`.",
```

Comment:
```suggestion
			"If the referenced key is not intended for the provider, please choose a different namespace from `testprovider:`.",
```
nit: makes it more specific